src/opm/parser/eclipse/share/keywords/000_Eclipse100/A/AQUANCON

```diff
@@ -8,7 +8,7 @@
  {"name" : "K1" , "value_type" : "INT"},
  {"name" : "K2" , "value_type" : "INT"},
  {"name" : "FACE" , "value_type" : "STRING"},
- {"name" : "INFLUX_COEFF" , "value_type" : "DOUBLE" , "dimension" : "Length*Length"},
+ {"name" : "INFLUX_COEFF" , "value_type" : "DOUBLE" , "dimension" : "Length*Length", "default_value" : -1.0},
```

Comment:
The INFLUX_COEFF item does not have a default value - according to the Eclipse documentation.

Reply:
It has a default value, i.e. the face area which I can only get from opm-grid. So, if you have any suggestions, am open to it.